publisher/publisher.go

```diff
@@ -89,43 +100,46 @@ func (pub *Impl) SubmitToCT(ctx context.Context, der []byte) error {
 	defer cancel()
 	chain := append([]ct.ASN1Cert{der}, pub.issuerBundle...)
 	for _, ctLog := range pub.ctLogs {
-		sct, err := ctLog.client.AddChainWithContext(localCtx, chain)
+		pub.stats.Inc("Submits", 1)
+		start := time.Now()
+		err := pub.singleLogSubmit(localCtx, chain, core.SerialToString(cert.SerialNumber), ctLog)
+		pub.stats.TimingDuration("SubmitLatency", time.Now().Sub(start))
```

Comment:
We should probably add some identifier to the CT log configuration and include it here so we actually know which logs are being slow.